classes/controller/FrontController.php

```diff
@@ -1239,14 +1227,14 @@ protected function getOverrideThemeDir()
      */
     public function getLayout()
     {
-        $layout = 'layout/layout-full-width.tpl';
+        $layout = 'layouts/layout-full-width.tpl';
 
         if ($this->display_column_left && $this->display_column_right) {
-            $layout = 'layout/layout-both-columns.tpl';
+            $layout = 'layouts/layout-both-columns.tpl';
         } elseif ($this->display_column_left) {
-            $layout = 'layout/layout-left-column.tpl';
+            $layout = 'layouts/layout-left-column.tpl';
         } elseif ($this->display_column_right) {
-            $layout = 'layout/layout-right-column.tpl';
+            $layout = 'layouts/layout-right-column.tpl';
```

Comment:
Wouldn't be better to use what you have in `$this->context->shop->theme->settings->available_layouts` ?